src/views/ScenarioManager/ScenarioManager.js

```diff
@@ -66,6 +79,26 @@ const ScenarioManager = (props) => {
     }
   }
 
+  function onScenarioRename(scenarioId, newScenarioName) {
+    if (scenarioId === currentScenario.id) {
+      setCurrentScenario({ name: newScenarioName });
+    }
+    renameScenario(WORKSPACE_ID, scenarioId, newScenarioName);
+  }
+
+  function checkScenarioNameValue(newScenarioName) {
+    if (newScenarioName.length === 0) {
+      return labels.scenarioRename.errors.emptyScenarioName;
+    } else {
+      if (newScenarioName.match(NAME_VALIDATOR) === null) {
+        return labels.scenarioRename.errors.forbiddenCharsInScenarioName;
+      } else if (ScenarioUtils.scenarioExistsInList(newScenarioName, scenarios)) {
+        return labels.scenarioRename.errors.existingScenarioName;
+      }
+    }
+    return null;
+  }
```

Comment:
This function should probably be defined in utils/ValidationUtils.js instead of importing `NAME_VALIDATOR` here.
Also, could you add a test with jest to check this function ? :smile: